src/vs/workbench/contrib/positronVariables/browser/components/variableItem.css

```diff
@@ -123,7 +123,12 @@
 	background-color: transparent;
 }
 
-.variable-item .details-column .right-column .viewer-icon:hover {
+.variable-item .details-column .right-column .viewer-icon.enabled:hover {
 	border: 1px solid var(--vscode-button-foreground, #ffffff);
-	background-color: var(--vscode-button-background, #0e639c);
+	background-color: var(--vscode-button-background, #b7c7d1);
+}
+
+.variable-item .details-column .right-column .viewer-icon.disabled {
+	cursor: auto;
+	opacity: 0.6;
```

Comment:
could we use something in place of opacity to communicate the disabled state? I worry that reducing the opacity may not be accessible with some themes and it may not be apparent that the view action is disabled.

Maybe for later, but we could have a different icon for when the action is queued up? Maybe the existing icon, but with a dot or clock in the corner to indicate that it's waiting/pending?